lib/internal/bootstrap/browser.js

```diff
@@ -114,14 +114,33 @@ function exposeGetterAndSetter(target, name, getter, setter = undefined) {
   });
 }
 
-// https://heycam.github.io/webidl/#Replaceable
-function defineReplacableAttribute(target, name, value) {
+// https://webidl.spec.whatwg.org/#Replaceable
```

Comment:
Interestingly the spec example says that `globalThis.hasOwnProperty('performance')` should evaluate to `false`, but all the browsers has it evaluate to `true`.

Reply:
Yeah.. I'll leave it as is. Probably we need to file an issue to whatwg/webidl.

Reply:
Opened https://github.com/whatwg/webidl/issues/1193 to track this issue.